tf-managed/modules/daily-snapshot/main.tf

```diff
@@ -75,10 +75,9 @@ resource "digitalocean_droplet" "forest" {
   graceful_shutdown = false
 
   connection {
-    host    = self.ipv4_address
-    user    = "root"
-    type    = "ssh"
-    timeout = "30m"
```

Comment:
why the timeout change?

Reply:
Fail fast and retry. If it doesn't finish successfully in a few minutes, given how trivial the initialisation is, it will likely not succeed in 30 minutes as well.